.github/workflows/release.yml

```diff
@@ -486,18 +349,18 @@ jobs:
         run: ls -R npm
       - name: Link optional dependencies
         run: pnpm install --no-frozen-lockfile
-        with:
+        # with:
           # this expects you to have a script called release which does a build for your packages and calls changeset publish
-          version: ${{ github.event.inputs.version }}
-          type: "release"
-          branch: ${{ github.ref_name }}
-          tools: "changeset"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-          REPOSITORY: ${{ github.repository }}
-          REF: ${{ github.ref }}
-          ONLY_RELEASE_TAG: true
+          # version: ${{ github.event.inputs.version }}
+          # type: "release"
+          # branch: ${{ github.ref_name }}
+          # tools: "changeset"
+        # env:
+          # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          # REPOSITORY: ${{ github.repository }}
+          # REF: ${{ github.ref }}
+          # ONLY_RELEASE_TAG: true
```

Comment:
This is broken on main

Reply:
ping @IWANABETHATGUY

Reply:
could you share the broken CI log link?

Reply:
This is tacked onto a pnpm run, maybe you meant `uses: modern-js-dev/actions@v2`.

You can take a look at the current Release runs, they are all failing.

Reply:
you are right